reflect.go

```diff
@@ -564,6 +567,10 @@ func (r *Reflector) reflectFieldName(f reflect.StructField) (string, bool, bool)
 		required = requiredFromJSONSchemaTags(jsonSchemaTags)
 	}
 
+	if r.PropertyNameConversion != nil {
```

Comment:
Hmmm, sorry for so many round trips, but it just occurred to me that unless the field has a `json:"<name>"` tag, changing the field in the schema isn't going to reflect the actual JSON, is it?

Reply:
It should work, I put the name conversion bit before the part which gets the name from the tag so that the name can still be overridden manually.

Also note, in the test the struct I am using does not have a tag.

I'll add a test case for testing the JSON output though.